app/code/Magento/Directory/Model/Currency/Import/AbstractImport.php

```diff
@@ -78,6 +78,17 @@ protected function _saveRates($rates)
         return $this;
     }
 
+    /**
+     * Creates array for provided currencies with empty rates.
+     *
+     * @param array $currenciesTo
+     * @return array
+     */
+    protected function _makeEmptyResponse(array $currenciesTo): array
```

Comment:
Why?

Reply:
This method was originally in FixerIo.php, moved it into this base class so that it can be reused by CurrencyConverterApi. But I just see the AbstractImport class is marked by `@api` annotation, so probably I can not add any new method in it. I can remove it if needed.

Reply:
@Hailong yep, please let this method be private as it was initially.

Reply:
I just reverted the changes in AbstractImport and FixerIo, but cloned makeEmptyResponse method in CurrencyConverterApi class.